dask/array/core.py

```diff
@@ -2841,7 +2841,7 @@ def from_array(
 
 
 def from_zarr(
-    url, component=None, storage_options=None, chunks=None, name=None, **kwargs
+    url, component=None, storage_options=None, chunks="auto", name=None, **kwargs
```

Comment:
The docstring entry for `chunks` should probably be updated to reflect this default

Reply:
What would you pass to retain the previous behaviour?

Reply:
You would get the chunking from the Zarr array and pass it in explicitly:

    chunks=my_zarr_array.chunks

Or if you wanted the smaller chunk sizes you would specify a smaller chunk size, perhaps in bytes

    chunks="1 MiB"

Reply:
This is also all the same logic that we currently have for any other dataset that defines a `chunks=` attribute.  I think that the default behavior is usually optimal.

Reply:
Sure, but it may be worthwhile indicating in the docstring how to get the inherent chunking.

Reply:
Can we keep `chunks=None` or something similar as an easy way to get the chunks on disk? It may not be easy to construct  the `my_zarr_array` if the only has a URL, say.

Reply:
We could.  Can I ask for additional motivation though?  We don't currently do this for HDF5 or NetCDF or any other format.  Why would we do this for Zarr?  Why do we care about the old behavior?  I expect that adding docs on this is just as likely to lead people astray as it is to help them.  

As a reminder, the automatic chunking is decently smart and we haven't ever gotten complaints about the choices that it makes, despite pretty heavy usage.  It will find a chunking that aligns with the existing chunking in storage, but is mildly larger in other dimensions if necessary.